snow/networking/handler/handler.go

```diff
@@ -740,9 +740,9 @@ func (h *handler) handleAsyncMsg(ctx context.Context, msg Message) {
 	h.asyncMessagePool.Send(func() {
 		if err := h.executeAsyncMsg(ctx, msg); err != nil {
 			h.StopWithError(ctx, fmt.Errorf(
-				"%w while processing async message: %s",
+				"%w while processing chan message: %s",
```

Comment:
I think this was correct before:
```suggestion
				"%w while processing async message: %s",
```